src/main.cpp

```diff
@@ -78,6 +78,33 @@ void usage()
   // clang-format on
 }
 
+/*
+  Return true and print a warning message if lockdown is enabled.
+*/
+static bool warn_on_lockdown(BPFfeature* feature)
```

Comment:
For what it's worth I sent https://lore.kernel.org/lkml/20200220151738.1492852-1-jcline@redhat.com/ so once it's merged you should be able to read /sys/kernel/security/lockdown for the current status as an unprivileged user.

Reply:
Ah nice. Bpftrace only supports running as root at the moment so we don't  really need it yet but good to have :)